app/models/registration_period.rb

```diff
@@ -21,4 +22,9 @@ def start_date_before_end_date
     errors
     .add(:start_date, "can't be after the end date") if start_date && end_date && start_date > end_date
   end
+
+  def presence_of_registration_ticket
+    errors
```

Comment:
instead of splitting this after the errors I would say it is more readable writing it like:

``` Ruby
if conference && conference.tickets.for_registration.empty?
  errors.add(:registration_ticket, "is not created. There should be at least one registration ticket.") 
end
```